CMakeLists.txt

```diff
@@ -172,8 +193,8 @@ set(MLIR_NPCOMP_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(MLIR_NPCOMP_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
 
 add_custom_target(check-npcomp)
-add_custom_target(check-all)
-add_dependencies(check-all check-npcomp)
+add_custom_target(check-npcomp-all)
```

Comment:
The `check-all` target seemed to conflict with the MLIR one. I added this `npcomp` namespacing, but I'm not sure if there is a better way.

Reply:
sgtm